### PR TITLE
Optimize Docker Container

### LINKED ISSUE #48: Контейнер неоптимизирован
Его можно уменьшить в размере, сократить количество слоёв.

### PR DESCRIPTION
**Pull request by cofob**
# Важный check-list

- [x] Я написал подробное описание моих изменений.
- [x] Я написал подробную причину моих изменений.
- [x] Я точно прочитал и использовал все пункты [этого списка](https://github.com/PerchunPak/pinger-bot/blob/master/CONTRIBUTING.md#перед-началом-работы).
- [x] Я записал изменение в `CHANGELOG.md` и добавил номер PR туда.
- [x] Изменения успешно проходят все этапы `make test`.

closes #48

### CODE REVIEW

File: Dockerfile
Comment (PerchunPak) [inline Dockerfile]: Why this duplicates?

File: Dockerfile
Comment (PerchunPak) [inline Dockerfile:9]: Please, use [recommended way to install the poetry](https://python-poetry.org/docs/master/#installation).

File: Dockerfile
Comment (PerchunPak) [inline Dockerfile:11]: Why `--without-hashes`?

File: docs/pages/docker.rst
Comment (PerchunPak) [inline docs/pages/docker.rst:104]: Add windows alternative please.

**Comment by cofob** [comment]
@PerchunPak add translations.

RU:
```
Запуск в podman настолько же прост как и в Docker. Всё сводиться к тому что вы заменяете во всех командах ``docker`` на ``podman``. Сложность может возникнуть только с автоматическим запуском, но podman предоставляет все необходимые инструменты для этого, с systemd это просто 1 команда для генерации юнита, запустите ``podman generate systemd --restart-policy always --name pingerbot > /etc/systemd/system/pingerbot.service``. После этого включите сервис ``systemctl daemon-reload && systemctl enable --now pingerbot``.
```
```
Начиная с версии 0.4.0 контейнер работает в rootless режиме, что означает, что если вы используете версию sqlite и внешний volume, вам необходимо предоставить права на папку. Это можно сделать с помощью команды ``chown -R 5000:500 <ваша папка>``.
```

**Review by PerchunPak — CHANGES_REQUESTED** [review]
Looks good, but I should look in it better.

File: Dockerfile
Comment (cofob) [reply Dockerfile]: Because of docker stupid useless ARG visibility levels

File: Dockerfile
Comment (cofob) [reply Dockerfile:9]: No. It requires curl. curl = more dependencies. More dependencies = larger docker image.

File: Dockerfile
Comment (cofob) [reply Dockerfile:11]: Because pip crashes with it

File: docs/pages/docker.rst
Comment (cofob) [reply docs/pages/docker.rst:104]: Those who host on Windows must die. It has no alternatives for chown as far as I know.

File: Dockerfile
Comment (PerchunPak) [reply Dockerfile:9]: So... squash it? `RUN apt-get install curl -y && curl .../install_poetry.py && apt-get remove curl -y`, even without it, curl will not go to final step, because you use `COPY --from`.

File: docs/pages/docker.rst
Comment (PerchunPak) [reply docs/pages/docker.rst:96]: https://serverfault.com/questions/27332/how-to-take-ownership-of-files-from-the-command-line

Comment (cofob) [reply docs/pages/docker.rst]: Docker for windows runs in wsl. WSL already have all essential commands for Linux.

File: Dockerfile
Comment (cofob) [reply Dockerfile:9]: curl install method recommended when you have multiple python versions. In docker we have only one python version, so this method is faster, than using a curl method.

File: Dockerfile
Comment (cofob) [reply Dockerfile:11]: Resolved?

File: Dockerfile
Comment (PerchunPak) [reply Dockerfile]: Anyway I changed it in https://github.com/PerchunPak/pinger-bot/pull/51/commits/ea7dfae3200652e2a3f24e75e5c394fb43a9b4a2.

File: docs/pages/docker.rst
Comment (PerchunPak) [reply docs/pages/docker.rst]: This auto handled by Docker on Windows, so I add not about this in https://github.com/PerchunPak/pinger-bot/pull/51/commits/aece8091cd9f2a1b073eb7fa3ea9158b5c66de3d.

**Review by cofob — COMMENTED** [review]
Incorrect. See DM.

**Review by PerchunPak — APPROVED** [review]
Okay, I think it's ready. @cofob, hoping I didn't break anything, and waiting for your `ok` before merge it to master.

**Review by cofob — COMMENTED** [review]
This is useless. Please remove.

Last step is always final. No need for this.

**Comment by PerchunPak** [comment]
> This is useless. Please remove.
> 
> Last step is always final. No need for this.

This is for pretty display during `docker build`:
image
Instead of
image

**Comment by cofob** [comment]
> Okay, I think it's ready. @cofob, hoping I didn't break anything, and waiting for your `ok` before merge it to master.

You can merge. Looks good.

**Pull request merged**
